doc/tech/approach.md

```diff
@@ -0,0 +1,99 @@
+
+The networking within an NFVI, fabric or otherwise, is an area where there is significant variability across implementations. Leaf-Spine topology is well established, however, after topology there are countless decisions an Operator needs to make. Differences arise from many aspects, for example, is the solution layer-2 or layer-3; is the routing static or dynamic; what mechanism is used for encapsulation; what mechanism is used for isolation; does it support SR-IOV; does it support DPDK; does it employ SmartNICs; does it employ distributed control or a centralized control driving a programmable fabric; and the list continues. The multitude of permutations enable NFVI architects (Operators and Suppliers) to design (or procure) a fabric/networking solution that's optimized for their needs, whether their needs are minimal, very extensive or somewhere in between.
+
+For CNTT, a strategy is needed that affords Operators the performance, flexibility, availability, maintainability and scalability their business requires, yet doesn't require OPNFV to design, manage and test prohibitive numbers of networking solutions.
+ 
+**Some points for CNTT to consider when contemplating recommendations:**
+   > * Despite large variances in implementation, the spectrum of networking capabilities ultimately delivered to Workloads is comparatively narrow
+   > * Standard CNTT methodology (i.e. normalize interfaces, APIs, capabilities and behaviors at the reference points) applies well to networking, helping mitigate the need to be overly prescriptive about implementation  
+   > * CNTT/OPNFV MUST provide a functional networking solution for the RI, and in support of RC  
+   > * CNTT/OPNFV potentially does NOT have to provide a production networking solution for RI  
+   > * CNTT/OPNFV is NOT planning to provide a production compute solution for RI
+
+
+<a name="5.1.1"></a>
+### 5.1.1 Executive Summary
+
+> _Placeholder for the Executive Summary (targeting Baldy for first draft). **If you are interested in writing this content, please contact the RM lead.**_
+
+<a name="5.1.2"></a>
+### 5.1.2 Networking Strategy Objectives
+
+This section catalogs CNTT's high-level objectives for the Networking and Fabric Strategy. 
+
+> _List needs to be prioritized; expect additional objectives to be added, as they arise. This represents the _What_, not the _How_.
+
+1. The implementation of Networking inside the HW Layer should not be visible to the VNF/CNF and should preferably not even be visible to the IaaS/CaaS
+1. Provide networks for L3 tenant, GWs, SDS, etc.
+1. Cleanly decouple interface/reference points between CNTT constituencies
+1. Provide interoperability at layer demarcation/reference points within the NFVI. Ex.:
```

Comment:
Suggest changing `NFVI` to `cloud infrastructure`

Reply:
done